linchpin/shell/__init__.py

```diff
@@ -91,10 +89,12 @@ def _handle_results(ctx, results, return_code):
 
     # PRINT OUTPUT RESULTS HERE
     ctx.log_state(output)
+    use_actual_rcs = json.loads(lpcli.get_cfg('lp',
+                                              'use_actual_rcs').lower())
```

Comment:
If you are using True/False, the proper use of this is to use ast.literaleval(value.title()), https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/linchpin/cli/__init__.py#L323 is a good example.